### PR TITLE
Increase Posts Text Limit to 1000

### PR DESCRIPTION
**Pull request by mendelB**
### What's this PR do?

This pull request increases the character limit for the Posts `text` field to 1000 in the database, and Post/Questionnaire validation rules.

### How should this be reviewed?
- create a post with more than 1000 characters without validation errors.

### Any background context you want to provide?
Our users seem interested in a higher character count! We've settled on 1000 as a logical limit.
Next up, we'll pry into Phoenix, and update the validations across the various text-specific submission actions

### Relevant tickets

References [Pivotal #178144452](https://www.pivotaltracker.com/story/show/178144452).

### Checklist

- [x] This PR has been added to the relevant Pivotal card.
- [ ] Documentation added for new features/changed endpoints.
- [ ] Added appropriate feature/unit tests.
- [ ] If new attributes were added to users, these changes are also surfaced in the admin interface for user profiles.
- [ ] If new attributes were added to users, then the data team already knows about these changes.

### CODE REVIEW

File: database/migrations/2021_07_22_171447_increase_text_character_count_in_posts_table_v2.php [hunk 1]
```diff
@@ -0,0 +1,32 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class IncreaseTextCharacterCountInPostsTableV2 extends Migration
```
Comment (mendelB) [inline database/migrations/2021_07_22_171447_increase_text_character_count_in_posts_table_v2.php:7]: This is the v2 to https://github.com/DoSomething/northstar/blob/f9bcd6343abb26b0b915f8f8f2e1e3c4831c8b46/database/migrations/2019_02_27_201348_increase_text_character_count_in_posts_table.php#L7

File: tests/Http/PostTest.php [hunk 2]
```diff
@@ -299,6 +299,7 @@ public function testCreatingAPostWithValidationErrors()
             'campaign_id' => 'dog', // This should be a numeric ID.
             'signup_id' => $signup->id, // This one is okay.
             'school_id' => 234, // This should be a string.
+            'text' => 'a' . str_repeat('h', 1012), // ahhh...hhhhh! (Exceeds 1000 character limit. (While on a roller coaster)).
             // and we've omitted the required 'type' and 'action' fields!
         ]);
```
Comment (DFurnes) [inline tests/Http/PostTest.php:305]: Let's add this field name to the `assertJsonValidationErrors` below!

**Review by DFurnes — APPROVED** [review]
📚

File: tests/Http/PostTest.php (hunk 2 above)
Comment (mendelB) [reply tests/Http/PostTest.php:305]: Lol I got too caught up on the comment and forgot about the code ☕  Updated in https://github.com/DoSomething/northstar/pull/1247/commits/aebaffae5bd933cf4977809599832c3248cc4d66

**Pull request merged**